<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRedliningTextEditor</class>
 <widget class="QWidget" name="QgsRedliningTextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>53</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolButtonBold">
     <property name="icon">
      <iconset resource="../../images/images.qrc">
       <normaloff>:/images/themes/default/textBold.png</normaloff>:/images/themes/default/textBold.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="spinBoxFontSize">
     <property name="suffix">
      <string> pt</string>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFontComboBox" name="fontComboBox"/>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="toolButtonItalic">
     <property name="icon">
      <iconset resource="../../images/images.qrc">
       <normaloff>:/images/themes/default/textItalic.png</normaloff>:/images/themes/default/textItalic.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QDoubleSpinBox" name="spinBoxRotation">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-179.900000000000006</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLineEdit" name="lineEditText">
     <property name="placeholderText">
      <string>Label text</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditText</tabstop>
  <tabstop>spinBoxFontSize</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>toolButtonBold</tabstop>
  <tabstop>toolButtonItalic</tabstop>
  <tabstop>spinBoxRotation</tabstop>
 </tabstops>
 <resources>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
